src/extension/background-script/SteganographyService.ts

```diff
@@ -47,7 +47,10 @@ const defaultOptions = {
 const isSameDimension = (dimension: Dimension, otherDimension: Dimension) =>
     dimension.width === otherDimension.width && dimension.height === otherDimension.height
 
-const getMaskBuf = memoizePromise((type: Mask) => downloadUrl(getUrl(`/image-payload/mask-${type}.png`)), undefined)
+const getMaskBuf = memoizePromise(
+    (type: Mask) => downloadUrl(getUrl(`/image-payload/mask-${type}.png`)).then((x) => x.arrayBuffer()),
```

Comment:
```suggestion
    (type: Mask) => (await downloadUrl(getUrl(`/image-payload/mask-${type}.png`))).arrayBuffer(),
```

Reply:
that's the same

Reply:
It seems weird to me that mixing `await` and `.then`.